docs/integration-playbook.md

```diff
@@ -0,0 +1,188 @@
+# Semgrep Integration Playbook
+
+This document describes common integration methods for including Semgrep in
+your workflows. This includes strategies for when and where to include Semgrep,
+advice on managing Semgrep in large codebases with many pre-existing issues,
+using Semgrep to help ensure the security of your organization, etc.
+
+Contents:
+
+* [Integration Points](#integration-points)
+  * [Development](#development)
+  * [Security](#security)
+* [Lifecycle Management](#lifecycle-management)
+  * [Installing Semgrep](#installing-semgrep)
+  * [Managing Rules](#managing-rules)
+
+## Integration Points
+
+This section describes common integration points where Semgrep is effective.
+
+### Development
+
+Integrating Semgrep into development workflows allows you to catch bugs and
+security issues before they reach production environments. There are many
+points where Semgrep can be integrated into development workflows. Generally
+speaking, integrating Semgrep earlier in the development pipeline allows you
+to catch mistakes sooner and easily fix any issues it surfaces. Correcting an
+issue in your code editor is much easier than waiting days or weeks for a
+security or QA team to surface the issue. However, CI systems are often more
+reliable and repeatable than results from developer's environments. This
+section will move from earliest to latest in the development pipeline.
+
+**One-off Analyses**
+
+Of course, Semgrep can always be [run locally](https://github.com/returntocorp/semgrep#getting-started)
+to investigate findings, search for new patterns, or simply to tinker with the
+tool. Running one-off analyses provides maximal flexbility while losing
+reliability, repeatablility, and consistency. We recommend using this approach
+during initial investigations and then moving to one of the below approaches
+once the tool has been tuned and [configured](https://github.com/returntocorp/semgrep/blob/develop/docs/configuration-files.md)
+to your environemnt.
+
+**In-editor**
```

Comment:
As part of the playbook, are there recommendations you have about what rules to run that are particularly good / fast for IDE? How should I think about using IDE with my devs?